php/sqlcommenter-php/packages/sqlcommenter-laravel/README.md

```diff
@@ -38,9 +43,22 @@ Add the following class above `Illuminate\Database\DatabaseServiceProvider::clas
     ...
 ]
 ```
-### Run unit tests
 
-Run unit tests using below command
+## Options
+
+With Laravel SqlCommenter, we have configuration to choose which tags to be appended to the comment. It is configurable in `config/google_sqlcommenter.php`
+
+| Field         | Included <br /> by default?                    | Description                                                                                                                 |
+| ------------- | ---------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| `action`      | <div style="text-align: center">&#10004;</div> | The [application namespace](https://laravel.com/docs/9.x/controllers) of the matching URL pattern in your routes/api.php    |
+| `controller`  | <div style="text-align: center">&#10004;</div> | The [name](https://laravel.com/docs/9.x/controllers) of the matching URL pattern as described in your routes/api.php        |
+| `db_driver`   | <div style="text-align: center">&#10004;</div> | The name of the php [database engine](https://laravel.com/docs/9.x/database)                                                |
+| `framework`   | <div style="text-align: center">&#10004;</div> | The word "laravel" and the version of laravel being used                                                                    |
+| `route`       | <div style="text-align: center">&#10004;</div> | The [route](https://laravel.com/docs/9.x/routing) of the matching URL pattern as described in your routes/api.php           |
+| `traceparent` | <div style="text-align: center">&#10004;</div> | The [W3C TraceContext.Traceparent field](https://www.w3.org/TR/trace-context/#traceparent-field) of the OpenTelemetry trace |
+
+## Run Unittests
+In the source repo, run unit tests using below command
```

Comment:
Lets either remove this unit test from this PR and checkin that or you have to create a complete "Development" section, which contains information on from where to pull the code (sqlcommenter monorepo), how to build the package, unit test, integration test, integrating the local library with PHP server.

Reply:
Updated